### PR TITLE
Deprecate GrainTracker::getElementalValues()

### PR DESCRIPTION
**Pull request by permcody**
This method is superseded by `getOpToGrainsVector()`. I plan to drop this method very soon (after all affected applications are cleaned up).

Ping @SudiptaBiswas, @dschwen

### CODE REVIEW

File: modules/phase_field/include/auxkernels/FeatureFloodCountAux.h [hunk 1]
```diff
@@ -36,6 +37,8 @@ class FeatureFloodCountAux : public AuxKernel
 
   /// Function being used to compute the value of this kernel
   const FeatureFloodCount & _flood_counter;
+  /// Extra interface is this is a GrainTracker object
```
Comment (dschwen) [inline modules/phase_field/include/auxkernels/FeatureFloodCountAux.h]: _if this is_ ?

File: modules/phase_field/src/auxkernels/FeatureFloodCountAux.C [hunk 2]
```diff
@@ -61,9 +65,18 @@ FeatureFloodCountAux::precalculateValue()
...
   case 5:  // ACTIVE_BOUNDS
-    _value = _flood_counter.getElementalValues(_current_elem->id()).size();
+    if (_grain_tracker_ptr)
+    {
+      const auto & op_to_grains = _grain_tracker_ptr->getOpToGrainsVector(_current_elem->id());
+      _value = std::count_if(op_to_grains.begin(), op_to_grains.end(),
+                             [](unsigned int grain_id)
+                             {
+                               return grain_id != libMesh::invalid_uint;
+                             });
```
Comment (dschwen) [inline modules/phase_field/src/auxkernels/FeatureFloodCountAux.C]: pretty

**Comment by permcody** [comment]
@idaholab/moose-team - Github has a new Review system built into their system. I think we should give it a whirl and see how it goes. I've turned it on in MOOSE so you'll see that merges are blocked now until a review is submitted. I need a review on this PR for instance since @dschwen is asleep at the wheel.

**Review by jwpeterson — APPROVED** [review]
I don't really understand the point of this.  If you thought we were having trouble getting PRs reviewed _before_, it's going to be even worse now that there is a formal system in place for writing a review/approving the PR beyond "click the green button."

**Comment by jwpeterson** [comment]
I'm also not exactly sure if the new approval system is working, even though I approved your PR, there is still a yellow circle saying "Required statuses must pass before merging" and I can't click the green button.

**Comment by jwpeterson** [comment]
I have no idea what "out-of-date" with base branch means... is this some new requirement?  I double-checked and verified that there are no conflicts between this branch and devel, so that's not the problem.

**Comment by jwpeterson** [comment]
> I have no idea what "out-of-date" with base branch means... is this some new requirement? I double-checked and verified that there are no conflicts between this branch and devel, so that's not the problem.

Turns out Cody had checked "Require branches to be up to date" box.  Unchecked that now...

**Pull request merged**